deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -23,6 +23,12 @@ spec:
       - image: gluster/gluster-centos:latest
         imagePullPolicy: IfNotPresent
         name: glusterfs
+        env:
+        - name: SYSTEMD_IGNORE_CHROOT
+          value: "1"
+        command:
+        - /usr/lib/systemd/systemd
+        - --system
```

Comment:
No I didn't. That doesn't say it's a bad idea, just that it isn't supported. Different things. :)